systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -360,7 +391,7 @@ synchronized void createBigAmountOfUsers(ExtensionContext extensionContext, Stri
 
     @BeforeAll
     void setup(ExtensionContext extensionContext) {
-        installClusterOperator(extensionContext, NAMESPACE);
+        installClusterWideClusterOperator(extensionContext, NAMESPACE, Constants.CO_OPERATION_TIMEOUT_DEFAULT, Constants.RECONCILIATION_INTERVAL);
```

Comment:
In case we use clusterWide CO, why are you mixing `NAMESPACE` and `namespaceName`?

Reply:
because:

1. `NAMESPACE` =  is the core namespace where CO is deployed, and @ParallelTest resources are deployed inside CO namespace
 2. `namespaceName` = otherwise